modules/web/src/assets/css/global/_main.scss

```diff
@@ -714,3 +714,101 @@ km-cni-version {
     margin-top: 30px;
   }
 }
+
+// side-nav-field content styling
+
+.collapse-sidenav {
+  overflow-x: hidden;
+  width: variables.$sidenav-width-collapsed !important;
+
+  .menu-item a i {
+    margin-right: 0;
+  }
+}
+
+.sub-menu-item {
+  position: relative;
+  z-index: 1;
+
+  a {
+    margin-left: 22px;
+    padding: 10px 5px 10px 30px;
+
+    &:hover {
+      border-radius: variables.$border-radius;
+    }
+  }
+
+  &.active > a::after {
+    content: '';
+    height: 60%;
+    left: 22px;
+    position: absolute;
+    top: 20%;
+    width: 3px;
+  }
+}
+
+km-side-nav-field .sub-menu-item:last-child {
+  padding-bottom: 10px;
```

Comment:
```suggestion
  margin-bottom: 10px;
```

To fix height of left border thing.

![screenshot-localhost_8000-2022 12 02-03_51_14](https://user-images.githubusercontent.com/13975988/205178315-9c31df47-54d5-4d34-84b1-61f3f6857595.png)

Reply:
margin will create another issue so i fix it with changing the height of `::after` for the last child and keep the padding